config-specs/paper/paper-world-defaults.yml

```diff
@@ -300,6 +300,10 @@ entities:
     phantoms-spawn-attempt-min-seconds:
       default: "60"
       description: Sets the minimum number of seconds between phantom spawn attempts
+    cooldown-failed-beehive-releases:
+      default: "true"
+      vanilla: "false"
```

Comment:
This goes into `config-specs/paper/vanilla/paper-world-defaults.yml`

Reply:
I don't think that exists anymore? it looks fine in the deploy too

Reply:
HUH I was tricked xD